### PR TITLE
feat: api to deposit to main account

### PR DESCRIPTION
**Pull request by vicimilenia**

### CODE REVIEW

**Review by sekarminati — APPROVED** [review]

File: api/customers/customers.go [hunk 1]
```diff
@@ -2,34 +2,72 @@ package customers
...
+	}
+
+	res, err := json.Marshal(result)
+	if err != nil {
+		helpers.HTTPError(w, http.StatusBadRequest, err.Error())
+		return
+	}
+
+	fmt.Fprintln(w, string(res))
+<<<<<<< HEAD
```
Comment (polipopoliko) [inline api/customers/customers.go]: please kindly delete this :)

File: api/customers/customers.go [hunk 2]
```diff
@@ -72,52 +110,16 @@ func (ch *CustomerHandler) Create(w http.ResponseWriter, r *http.Request){
...
 		helpers.HTTPError(w, http.StatusBadRequest, "Email Token Failed")
-		return 
+		return
 	}
 }
-
-type GetProfileResult struct {
-	Customers models.Customers `json:"customers"`
-	Accounts  models.Accounts  `json:"accounts"`
+=======
```
Comment (polipopoliko) [inline api/customers/customers.go]: please kindly remove this :)

File: api/customers/customers.go [hunk 3]
```diff
@@ -72,52 +110,16 @@ func (ch *CustomerHandler) Create(w http.ResponseWriter, r *http.Request){
...
-	res, err := json.Marshal(result)
-	if err != nil {
-		helpers.HTTPError(w, http.StatusBadRequest, err.Error())
-		return
-	}
-
-	fmt.Fprintln(w, string(res))
-}
\ No newline at end of file
+>>>>>>> develop
```
Comment (polipopoliko) [inline api/customers/customers.go]: please kindly remove this :)

File: api/customers/deposit.go [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+			AccountNum:  request.AccountNumber,
+			DestAccount: request.AccountNumber,
+			TranAmount:  request.BalanceAdded,
+			Description: constants.Deposit,
+			CreatedAt:   time.Now(),
+		}
+
+		err = trx.LogTransaction(log)
+		if err != nil {
+			fmt.Println(err)
```
Comment (polipopoliko) [inline api/customers/deposit.go:87]: is it still needed?

File: api/customers/customers.go [hunk 5]
```diff
@@ -62,6 +62,7 @@ func (ch *CustomerHandler) GetProfile(w http.ResponseWriter, r *http.Request) {
 	}
 
 	fmt.Fprintln(w, string(res))
+<<<<<<< HEAD
```
Comment (yulyulyharuka) [inline api/customers/customers.go]: delete this string. Nanti gabisa di compile

File: api/customers/customers.go [hunk 6]
```diff
@@ -119,3 +120,6 @@ func (ch *CustomerHandler) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+=======
+}
+>>>>>>> develop
```
Comment (yulyulyharuka) [inline api/customers/customers.go]: delete this string. Nanti gabisa di compile

**Review by yulyulyharuka — CHANGES_REQUESTED** [review]

**Review by polipopoliko — APPROVED** [review]

**Review by yulyulyharuka — APPROVED** [review]

**Comment by polipopoliko** [comment]
Done merging using cli

**Pull request closed without merging**